Imakefile

```diff
@@ -7,7 +7,9 @@ OPTION =
 # If function dosen't find , add  -lsocket or whatever you need .
 # This is example .
 #ADD_LIBRARIES = -lxpg4 -lsocket -lnsl
-ADD_LIBRARIES = -lxpg4
+ifeq ($(uname -s), Solaris)
```

Comment:
If you know more about `lxpg4` it would be great I only found it in solaris context which is why I added the condition like this.